internal/lib/dynamic_client.go

```diff
@@ -202,6 +203,30 @@ func GetBootstrapCRData() (string, string, string) {
 	return secretName, secretNamespace, userName
 }
 
+func GetNetinfoCRData() map[string]string {
+	lslrMap := make(map[string]string)
+	dynamicClient := GetVCFDynamicClientSet()
+	crdClient := dynamicClient.Resource(NetworkInfoGVR)
+	crList, err := crdClient.List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		utils.AviLog.Errorf("Error getting Networkinfo CR %v", err)
+		return lslrMap
+	}
+
+	for _, obj := range crList.Items {
+		spec, found, err := unstructured.NestedStringMap(obj.UnstructuredContent(), "topology")
+		if found && err == nil {
+			lr := spec["gatewayPath"]
```

Comment:
Is it possible that the keys don't exist for a particular instance of this CRD? Let's say an older version?